Fst2List.cpp

```diff
@@ -1904,6 +2319,24 @@ void CFstApp::printPathNames(U_FILE *f) {
   }
 }
 
+void CFstApp::setGrammarMode(char* fst2_filename) {
+  char* tmp = (char*)malloc(sizeof(char) * strlen(fst2_filename));
```

Comment:
Instead of `tmp` use a self-describing variable name

Reply:
done